carla-carma-integration/carla-carma-vehicle/src/carla_carma_vehicle/carla_carma_vehicle.py

```diff
@@ -62,11 +62,12 @@ def __init__(self):
         self.player = None
         self.player_created = False
         self.sensor_actors = []
-        self.actor_filter = rospy.get_param('~vehicle_filter', 'vehicle.*')
+
+        #Get vehicle parameter values
+        self.actor_filter = rospy.get_param('/vehicle_model', 'vehicle.*')
```

Comment:
Not sure that this switch makes sense. I'm not totally sure what the actor filter is, but the original code appears to be treating it as a different thing from the vehicle model, so I'd be inclined to leave it that way until we have a clear understanding of what exactly it's doing. If you do have that understanding though and I'm the mistaken one here, we can proceed.